packages/stacks-classic/lib/components/empty-state/empty-state.visual.test.ts

```diff
@@ -1,13 +1,14 @@
 import { html } from "@open-wc/testing";
-import { SpotEmptyXL } from "@stackoverflow/stacks-icons-legacy";
+import { SpotEmpty } from "@stackoverflow/stacks-icons/spots";
 import { runVisualTests } from "../../test/visual-test-utils";
 import "../../index";
 
 describe("empty-state", () => {
     runVisualTests({
         baseClass: "s-empty-state",
         children: {
-            default: `${SpotEmptyXL}<p class="mt24"><strong>Hello!</strong> This is a wonderful empty state component.</p>`,
+            default: `${SpotEmpty.replace("svg-spot", "svg-spot native")}
+            <p>This is a wonderful empty state component.</p>`,
```

Comment:
I recommend adding the `.s-empty-state--title` child element here or as an additional variation within `children`